ethers-providers/src/transports/quorum.rs

```diff
@@ -506,6 +506,49 @@ where
                 let value = serde_json::to_value(block).expect("Failed to serialize U64");
                 Ok(serde_json::from_value(value)?)
             }
+            "eth_sendTransaction" | "eth_sendRawTransaction" => {
+                // non-idempotent requests may fail due to delays in processing even though the
+                // operation was a success.
+
+                // TODO: be more clever than to just accept any Ok response and to look for
```

Comment:
Fwiw because these error messages aren't in the RPC spec (and EVM compatible chains have an awful tendency of dealing with errors in their own way with RPCs) I'd probably recommend against putting this in here in the future. But if we see the need I'd ofc be open to some more clever logic like you suggest

Reply:
That does make things difficult. I will remove the todo for now.